ui/src/app/pages/workspace/featured-workspaces.tsx

```diff
@@ -0,0 +1,315 @@
+        These workspaces demonstrate how computable electronic phenotypes can be
+        implemented within the <AoU /> dataset using examples of previously
+        published phenotype algorithms. You can open the workspaces to view them
+        or “duplicate” the workspaces to edit and execute the algorithms.
+      </div>
+    ),
+    icon: phenotypeLibrary,
+    category: FeaturedWorkspaceCategory.PHENOTYPE_LIBRARY,
+  },
+  TUTORIAL_WORKSPACES: {
+    title: 'Tutorial Workspaces',
+    description: (
+      <div>
+        These workspaces provide instructions for key Researcher Workbench
+        components and representations of the <AoU /> dataset. You can open the
+        workspaces to view them or “duplicate” the workspaces to edit and
+        execute them.
+      </div>
+    ),
+    icon: tutorialWorkspaces,
+    category: FeaturedWorkspaceCategory.TUTORIAL_WORKSPACES,
+  },
+  DEMO_PROJECTS: {
+    title: 'Demonstration Projects',
+    description: (
+      <div>
+        Workspaces included here are meant to demonstrate end-to-end analyses
+        that can be performed using <AoU /> Data. The current set of research
+        example analyses were developed as part of <AoU /> Demonstration
+        Projects. They are aimed at demonstrating the quality, utility, and
+        diversity of <AoU /> data by replicating previously published studies.
+        You can open the workspaces to view them or “duplicate” the workspaces
+        to edit and execute them.
+      </div>
+    ),
+    icon: demonstration,
+    category: FeaturedWorkspaceCategory.DEMO_PROJECTS,
+  },
+  COMMUNITY: {
+    title: 'Community Workspaces',
+    description: (
+      <div>
+        Community workspaces are curated and maintained by researchers using the
+        Researcher Workbench and are intended for use by the Researcher
+        Workbench community. Please be sure to review the 'About' section of the
+        workspace for how to use and cite the workspace in your own research and
+        who to contact with any questions about the workspace.
+      </div>
+    ),
+    icon: community,
+    category: FeaturedWorkspaceCategory.COMMUNITY,
+  },
+};
+
+interface TabProps {
+  title: string;
+  icon: string;
+  onClick: () => void;
+  selected: boolean;
+}
+
+const LibraryTab = ({ title, icon, onClick, selected }: TabProps) => {
+  return (
+    <Clickable
+      style={
+        selected
+          ? { ...styles.menuLink, ...styles.menuLinkSelected }
+          : styles.menuLink
+      }
+      onClick={onClick}
+      hover={styles.menuLinkSelected}
+    >
+      <img src={icon} style={styles.iconStyling} />
+      {title}
+    </Clickable>
+  );
+};
+
+export const FeaturedWorkspaces = (props) => {
+  const [currentTab, setCurrentTab] = useState<LibraryTabProps>(
+    libraryTabs.TUTORIAL_WORKSPACES
+  );
+  const [errorText, setErrorText] = useState<string>('');
+
+  const [workspaceList, setWorkspaceList] = useState<Array<WorkspaceResponse>>(
```

Comment:
optional nitpick: anywhere you can use `Array<Whatever>` you can instead use `Whatever[]` -  it means the same thing.